docs/test-plan.md

```diff
@@ -0,0 +1,248 @@
+# Test Plan: lockbox-extension
+
+## Ownership
+
+* Product Integrity: Matt Brandt
+* Product Manager: Sandy Sage
+* Engineering Manager: Devin Reams
+* Engineering Leads: Matt Miller & Jim Porter
+
+## Purpose
+
+Test scope for Q3 and Q4 of the Lockbox Firefox extension. Q3 will take the project through the internal Alpha phase of the product to Q4 when the team will tune a product that we can ship as a limited Beta to external users via Test Pilot.
+
+### Entry Criteria
+
+* QA has access to all PRDs, mocks, and related documents.
+* A run script that installs the WebExtension for Firefox Nightly
+* A compiled and signed extension for Firefox Beta and Release
+* Staging and release channels
+
+### Exit Criteria
+
+* All bugs against the features have been triaged
+* All P1 and P2 bugs have been resolved fixed and verified by Product Integrity
```

Comment:
Are we tagging issues w/ priorities? Or should this just say all [**TxP**](https://github.com/mozilla-lockbox/lockbox-extension/issues?q=is%3Aopen+is%3Aissue+label%3ATxP) tagged issues?

Reply:
Good catch, when this doc was authored, TxP did not exist.